### DISCUSSION TITLE
Does quick.js not have an API for creating files?

### DISCUSSION THREAD

**Discussion by Ray-D-Song** [quickjs-ng/quickjs#523] [Q&A] [answered]
I saw the mkdir API used to create folders, but I didn't find the content for creating files.

**Comment by saghul** [comment, marked as answer]
Check out std.open. There are examples in the tests.

**Reply by Ray-D-Song** [reply]
Thank you, because I am not very familiar with the C Library. Thank you for solving my problem.
